### PR TITLE
Feature/package tar artifacts

### LINKED ISSUE #2750: Deb don't   support the specified directory installation
sudo dpkg -i --instdir=/data nebula-graph-2.0.0.ubuntu1604.amd64.deb 安装失败
image

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Close #2750 

Package tar and upload to oss.

### CODE REVIEW

File: package/package.sh [hunk 1]
```diff
@@ -206,10 +210,13 @@ function dump_syms {
 }
 
 # The main
-build $version $enablesanitizer $static_sanitizer $build_type $branch
-rm -rf ${build_dir}/src
+build $version $enablesanitizer $static_sanitizer $build_type $branch "OFF" "/usr/local/nebula"
```
Comment (CPWstatic) [inline package/package.sh:213]: Why not use `install_prefix`?

File: package/package.sh [hunk 2]
```diff
@@ -206,10 +210,13 @@ function dump_syms {
...
 package $strip_enable
 if [[ $dump_symbols == ON ]]; then
     echo ">>> start dump symbols <<<"
     dump_syms
 fi
+
+# tar package
+build $version $enablesanitizer $static_sanitizer $build_type $branch "ON" "/"
```
Comment (yixinglu) [inline package/package.sh:221]: install the tar package into root directory by default?

Comment (Shylock-Hg) [reply package/package.sh:221]: '/' means root of tar package here.

File: package/package.sh (hunk 1 above)
Comment (Shylock-Hg) [reply package/package.sh:213]: That will modified in different stage.

**Review by yixinglu — COMMENTED** [review]
LGTM

**Review by czpmango — APPROVED** [review]

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -33,8 +33,7 @@ jobs:
         env:
           SHA_EXT: sha256sum.txt
         run: |
-          filename=$(find pkg-build/cpack_output -type f \( -iname \*.deb -o -iname \*.rpm \))
```
Comment (darionyaphet) [inline .github/workflows/nightly.yml:36]: seems merge calculate the SHA check sum into one line?

Comment (Shylock-Hg) [reply .github/workflows/nightly.yml:36]: Yes

**Review by CPWstatic — APPROVED** [review]

**Pull request merged**